apps/openmw/mwmechanics/character.cpp

```diff
@@ -914,7 +914,7 @@ void CharacterController::handleTextKey(const std::string &groupname, const std:
         return;
     }
     if(evt.compare(0, 10, "soundgen: ") == 0
-     && evt.compare(10, evt.size()-10, "land") != 0) // Morrowind ignores land soundgen for some reason
+     && evt.compare(10, 4, "land") != 0) // Morrowind ignores land soundgen for some reason
```

Comment:
This will catch anything that starts with "land", including "landing" or whatever else is possible for a soundgen entry.